technische-universitat-dresden-historische-musikwissenschaft-note.csl

```diff
@@ -28,8 +28,8 @@
       <term name="issue" form="short">H.</term>
       <term name="translator" form="verb-short">deutsch von</term>
       <term name="presented at">gehalten auf</term>
-      <term name="no date" form="short">o.&#8239;J.</term>
-      <term name="et-al">u.&#8239;a.</term>
+      <term name="no date" form="short">o. J.</term>
+      <term name="et-al">u. a.</term>
```

Comment:
@rmzelle, thanks, good to know. I use the visual csl editor a lot, it automatically converts decimal code to special characters and I always thought this was a good thing. I will avoid this in the future. For now, could you merge anyway and have your script do the dirty work?